### ISSUE TITLE
The reference training scripts in training vgg16_bn

### ISSUE THREAD

**Issue by PkuDavidGuan** [pytorch/vision#4086]
## To Reproduce

When I run `train.py` to train `vgg16_bn` in https://github.com/pytorch/vision/tree/master/references/classification, I follow the `README.md` to use the following script:
```shell
CUDA_VISIBLE_DEVICES=0,1,2,3 \
python train.py \
  -b	32 \
  --epochs	90 \
  --lr	0.1 \
  --momentum	0.9 \
  --wd 1e-4 \
  --lr-step-size	30 \
  --lr-gamma	0.1 \
  --model vgg16_bn \
```

However, I get nan loss in the training:
```shell
Epoch: [0]  [    0/40037]  eta: 1 day, 22:47:18  lr: 0.1  img/s: 11.54904442989675  loss: 7.2017 (7.2017)  acc1: 0.0000 (0.0000)  acc5: 0.0000 (0.0000)  time: 4.2071  data: 1.4362  max mem: 7117
Epoch: [0]  [   10/40037]  eta: 6:56:59  lr: 0.1  img/s: 120.38295738808868  loss: 22.6492 (28.1895)  acc1: 0.0000 (0.0000)  acc5: 0.0000 (0.5682)  time: 0.6251  data: 0.1307  max mem: 7117
Epoch: [0]  [   20/40037]  eta: 5:02:48  lr: 0.1  img/s: 120.64070267972147  loss: 19.6099 (22.1551)  acc1: 0.0000 (0.0000)  acc5: 0.0000 (0.7440)  time: 0.2664  data: 0.0001  max mem: 7117
Epoch: [0]  [   30/40037]  eta: 4:22:16  lr: 0.1  img/s: 120.31022672165666  loss: 8.9287 (18.1558)  acc1: 0.0000 (0.0000)  acc5: 0.0000 (0.8065)  time: 0.2659  data: 0.0001  max mem: 7117
Epoch: [0]  [   40/40037]  eta: 4:01:21  lr: 0.1  img/s: 121.3082277148169  loss: 18.1619 (5291612553.3549)  acc1: 0.0000 (0.0000)  acc5: 0.0000 (0.6860)  time: 0.2655  data: 0.0001  max mem: 7117
Epoch: [0]  [   50/40037]  eta: 3:47:57  lr: 0.1  img/s: 124.05593425719074  loss: 216951619584.0000 (nan)  acc1: 0.0000 (0.0000)  acc5: 0.0000 (0.6740)  time: 0.2625  data: 0.0001  max mem: 7117
Epoch: [0]  [   60/40037]  eta: 3:38:44  lr: 0.1  img/s: 123.1053751053641  loss: nan (nan)  acc1: 0.0000 (0.0512)  acc5: 0.0000 (0.6148)  time: 0.2591  data: 0.0001  max mem: 7117
Epoch: [0]  [   70/40037]  eta: 3:32:07  lr: 0.1  img/s: 123.92169404986682  loss: nan (nan)  acc1: 0.0000 (0.0440)  acc5: 0.0000 (0.5282)  time: 0.2583  data: 0.0001  max mem: 7117
Epoch: [0]  [   80/40037]  eta: 3:27:10  lr: 0.1  img/s: 123.72772065933924  loss: nan (nan)  acc1: 0.0000 (0.0386)  acc5: 0.0000 (0.4630)  time: 0.2586  data: 0.0001  max mem: 7117
Epoch: [0]  [   90/40037]  eta: 3:23:17  lr: 0.1  img/s: 123.27316646200391  loss: nan (nan)  acc1: 0.0000 (0.0343)  acc5: 0.0000 (0.4121)  time: 0.2588  data: 0.0001  max mem: 7117
Epoch: [0]  [  100/40037]  eta: 3:20:11  lr: 0.1  img/s: 123.82109016435032  loss: nan (nan)  acc1: 0.0000 (0.0309)  acc5: 0.0000 (0.5260)  time: 0.2589  data: 0.0001  max mem: 7117
```

## Environment

```
Collecting environment information...
PyTorch version: 1.7.1+cu101
Is debug build: False
CUDA used to build PyTorch: 10.1
ROCM used to build PyTorch: N/A

OS: Ubuntu 16.04.6 LTS (x86_64)
GCC version: (Ubuntu 5.4.0-6ubuntu1~16.04.12) 5.4.0 20160609
Clang version: Could not collect
CMake version: Could not collect
Libc version: glibc-2.10

Python version: 3.7 (64-bit runtime)
Python platform: Linux-4.4.0-131-generic-x86_64-with-debian-stretch-sid
Is CUDA available: True
CUDA runtime version: Could not collect
GPU models and configuration:
GPU 0: TITAN Xp
GPU 1: TITAN Xp
GPU 2: TITAN Xp
GPU 3: TITAN Xp
GPU 4: TITAN Xp
GPU 5: TITAN Xp

Nvidia driver version: 418.56
cuDNN version: /usr/local/cuda-9.0/lib64/libcudnn.so.7
HIP runtime version: N/A
MIOpen runtime version: N/A

Versions of relevant libraries:
[pip3] numpy==1.19.5
[pip3] torch==1.7.1+cu101
[pip3] torchaudio==0.7.2
[pip3] torchfile==0.1.0
[pip3] torchnet==0.0.4
[pip3] torchvision==0.8.2+cu101
[conda] blas                      1.0                         mkl    defaults
[conda] cudatoolkit               10.1.243             h6bb024c_0    defaults
[conda] mkl                       2020.2                      256    defaults
[conda] mkl-service               2.3.0            py37he904b0f_0    defaults
[conda] mkl_fft                   1.1.0            py37h23d657b_0    defaults
[conda] mkl_random                1.1.1            py37h0573a6f_0    defaults
[conda] numpy                     1.19.5                   pypi_0    pypi
[conda] torch                     1.7.1+cu101              pypi_0    pypi
[conda] torchaudio                0.7.2                    pypi_0    pypi
[conda] torchfile                 0.1.0                    pypi_0    pypi
[conda] torchnet                  0.0.4                    pypi_0    pypi
[conda] torchvision               0.8.2+cu101              pypi_0    pypi
```

## Additional Context
I guess the learning rate is too high for `vgg16_bn`, since I decrease the `lr=0.01` and the nan loss is gone. However, the `README.md` said:
> Here `$MODEL` is one of `alexnet`, `vgg11`, `vgg13`, `vgg16` or `vgg19`. Note that `vgg11_bn`, `vgg13_bn`, `vgg16_bn`, and `vgg19_bn` include batch normalization and thus are trained with the default parameters.

Therefore, I wonder what is the default training script for `vgg16_bn` because the default `lr=0.1` is wrong for `vgg16_bn`.

**Comment by NicolasHug** [comment]
Hi @PkuDavidGuan , thanks for the report.

The issue here is probably that `python train.py` doesn't actually run on multiple GPUs, it just uses 1. This has an effect of the batch size, as `-b` refers to the batch size *per GPU*, and thus also has an effect on the ideal learning rate.

A more correct command would be to use e.g.
```
python -m torch.distributed.launch --nproc_per_node=4 --use_env train.py -b  32 \
  --epochs      90 \
  --lr  0.1 \
  --momentum    0.9 \
  --wd 1e-4 \
  --lr-step-size        30 \
  --lr-gamma    0.1 \
  --model vgg16_bn
```

where `--nproc_per_node=4` indicates the number of GPUs. Here, the default lr = 0.1 seems OK and the loss is properly decreasing. I'll submit a PR to change the instructions in the REAMDE.
